packages/x-components/src/adapter/mocked-responses.ts

```diff
@@ -23,6 +23,7 @@ export const getPopularSearchesEndpoint = `${mockedApiUrl}/popular-searches`;
 export const getNextQueriesEndpoint = `${mockedApiUrl}/next-queries`;
 export const searchEndpoint = `${mockedApiUrl}/search`;
 export const trackEndpoint = `${mockedApiUrl}/track`;
+export const getSemanticQueriesEndpoint = `${mockedApiUrl}/semantic-queries`;
```

Comment:
There is no `semantic-queries` mockedResponse using `createSemanticQuery`, is this ok?

Reply:
Added 👍